doc/whats_new.rst

```diff
@@ -77,6 +77,8 @@ Bug
 
 - Fix :func:`mne.Evoked.decimate` not setting ``inst.first`` and ``inst.last`` properly by `Marijn van Vliet`_
 
+- Fix :func:`mne.io.brainvision.read_raw_brainvision` not handling ``Event`` markers created by PyCorder correctly by `Richard Höchenberger`_
```

Comment:
```suggestion
- Fix :func:`mne.io.read_raw_brainvision` not handling ``Event`` markers created by PyCorder correctly by `Richard Höchenberger`_
```

Reply:
although both are valid namespace paths, the documentation cross-referencing relies on [a specific document](https://github.com/mne-tools/mne-python/blob/master/doc/python_reference.rst) that defines the cross-referencable subset of the namespace.